aiida/cmdline/commands/cmd_storage.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""`verdi storage` commands."""
+
+import click
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+from aiida.cmdline.params import options
+from aiida.cmdline.utils import echo
+from aiida.common import exceptions
+
+
+@verdi.group('storage')
+def verdi_database():
+    """Inspect and manage the internal storage."""
+
+
+@verdi_database.command('migrate')
+@options.FORCE()
+def backend_migrate(force):
```

Comment:
I think the command functions should just take the name `storage_{command}` where `{command}` is the subcommand.
```suggestion
def storage_migrate(force):
```